### ISSUE TITLE
Brotato (1942280)

### ISSUE THREAD

**Issue by sobran** [ValveSoftware/Proton#7898]
# Compatibility Report
- Name of the game with compatibility issues: Brotato
- Steam AppID of the game: 1942280

## System Information
- GPU: RTX 4080 Super
- Video driver version: nvidia 555.58.02
- Kernel version: 6.9.9-arch1-1
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/sobran/6db343f4ca94448a0c49fe2ae3bb4565#file-system_info
- Proton version: Experimental

## I confirm:
- [X] that I haven't found an existing compatibility report for this game.
- [X] that I have checked whether there are updates for my system available.

[steam-1942280.log](https://github.com/user-attachments/files/16203796/steam-1942280.log)
![brotato](https://github.com/user-attachments/assets/cf7c50b1-f05d-4e5a-8d87-141d6dd6c0b0)

## Symptoms 

When I start Brotato, it initially launches in windowed mode and then automatically switches to fullscreen mode. This behavior is consistent with Windows 11 and seems normal. However, under Linux with KDE, the game only occupies a small portion of the screen once it switches to fullscreen, and the game interface is not fully displayed. This makes it impossible for me to navigate the menu to disable fullscreen mode in the game options.

I'm not sure if this issue is with Proton or KDE.

## Reproduction

1. Use KDE Plasma 6.1.2 with the aforementioned nvidia driver 
2. Start Brotato

**Comment by Leopard1907** [comment]
I would say you should check if that is some sort of driver WSI issue or not.
Install this package.

https://archlinux.org/packages/extra/x86_64/vulkan-swrast/

After that launch the game with `VK_DRIVER_FILES=/usr/share/vulkan/icd.d/lvp_icd.x86_64.json DXVK_HUD=full %command%` and see if it behaves same still.

Hud part is just to confirm to see if it loads with lavapipe or not.

**Comment by sobran** [comment]
If I start the game  the way you suggested, the game behaves exactly as before.

However, I have found two workarounds.

1. Start the game with "gamescope -W 2560 -H 1440 -r 360 -f %command%". After that, I can see the full interface of the game. However, Gamescope itself is not running in fullscreen. I then need to bring the window to fullscreen using KWin.

2. Start the game without any arguments. However, first create a KWin window rule that automatically brings windows with the title "Brotato" to fullscreen.

Both solutions work independently of each other.

**Comment by glenni83** [comment]
Have the same issue on my system. 

I was playing a bit yesterday after finding a "workaround", where i tab to taskmanager with the game up. click on application and remove fullscreen. Then it goes in a fullscreen over my taskbar. Go into the game, remove the fullscreen ingame, and go out down on the application again and press fullscreen. Then  the game does proper fullscreen.

**Comment by glenni83** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/1937#issuecomment-2282891273

Tested. It works here :) Thanks

**Comment by abud7eem** [comment]
from my testing this only happens on wayland on both kde/gnome
@kisak-valve can you pass it to the team ty

**Comment by simifor** [comment]
I tried this game on kde xorg and wayland with proton 9, experimental and bleeding edge. The behavior was the same across the board, it scaled fine. Tested with plasma 6.2.3 and xorg-xwayland 24.1.4, it would be interesting to know more about the current setup of those affected.

**Comment by Milli189** [comment]
yeah I've just encountered the same issue, using proton 8 makes it look centered again, but it's not optimal to use proton 8

**Comment by Milli189** [comment]
the other solution I've just found is to use proton-ge, I don't have the issue with it.

Also the wayland driver with proton-ge works fine and doesen't have the issue, you can activate it with this launch option: PROTON_ENABLE_WAYLAND=1 %command%

**Comment by Milli189** [comment]
the proton versions that are affected by this regressions, and it's a regression since proton 8.0-4 doesen't have this issue, are proton 9.0-4 and newer.

**Comment by Milli189** [comment]
another thing that I've just discovered is that proton 10 release candidate 2 and proton hotfix don't have this issue.

so the only two versions of proton that have this issue are proton 9.0-4 and proton experimental
